src/main/java/org/springframework/integration/kafka/inbound/KafkaMessageDrivenChannelAdapter.java

```diff
@@ -297,7 +299,7 @@ public int afterShutdown() {
 		}
 
 		@Override
-		public void onMessage(ConsumerRecord<K, V> record, Acknowledgment acknowledgment) {
+		public void onMessage(ConsumerRecord<K, V> record, Acknowledgment acknowledgment, Consumer<?, ?> consumer) {
```

Comment:
Yes; we probably should, but that would be in spring-kafka (where the message mapping is done). I'll do it over there.